test/pom.xml

```diff
@@ -76,6 +76,11 @@
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+            <version>${testcontainers.version}</version>
+        </dependency>
```

Comment:
This dependency should be `optional`. We don't want to transitively force TestContainers on users.

Reply:
good catch, updated!